src/OpenTelemetry.Instrumentation.Wcf/OpenTelemetry.Instrumentation.Wcf.csproj

```diff
@@ -23,10 +23,10 @@
     <Reference Include="System.ServiceModel" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardMinimumSupportedVersion)' OR '$(TargetFramework)' == 'net6.0' " >
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardMinimumSupportedVersion)' OR '$(TargetFramework)' == '$(NetMinimumSupportedVersion)' " >
```

Comment:
I would switch this to detect `.NETCoreApp` so it doesn't need adjustment if some other target (eg `$(NetMinimumSupportedVersion);net9.0`) is added.

```xml
  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardMinimumSupportedVersion)' OR '$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
    <PackageReference Include="System.ServiceModel.Primitives" Version="4.7.0" />
    <PackageReference Include="System.Drawing.Common" Version="4.7.2" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
  </ItemGroup>
```

But also consider bumping to `8.0.0` versions for `net8.0`+:

```xml
  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardMinimumSupportedVersion)'">
    <PackageReference Include="System.ServiceModel.Primitives" Version="4.7.0" />

    <!-- System.Security.Cryptography.Xml is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-rxg9-xrhp-64gj -->
    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
    <PackageReference Include="System.ServiceModel.Primitives" Version="8.0.0" />

    <!-- System.Security.Cryptography.Xml is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-555c-2p6r-68mm -->
    <PackageReference Include="System.Security.Cryptography.Xml" Version="8.0.2" />
  </ItemGroup>
```

The `8.0.0` versions don't bring in `System.Drawing.Common` at all. The current form will force `System.Drawing.Common` onto users who may have already bumped to `8.0.0` to eliminate it 🤔

Reply:
First part done. Second deferred to #2274